### PR TITLE
Update Guides structure

### PR DESCRIPTION
**Pull request by Tabrarian**
Remove redundant directory, consolidate Guides landing page, create landing page for Contributing.

### CODE REVIEW

File: _docs/guides/contributing.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Contributing to Forseti
+order: 001
```
Comment (mirons-google) [inline _docs/guides/contributing.md]: This will need to be 003

**Comment by Tabrarian** [comment]
I updated order to address conflicts, but set it so contributing is above testing. Since we only have 2 pages, I don't think we need to create a sub-directory at this time, but will keep in mind if we add more content later. It looks like it would be good to add sub-directories to the Howtos though; I'll start a separate PR for that.

**Comment by carise** [comment]
Sorry for not responding sooner on this PR. Could you resolve the merge conflict on the files noted above?

**Comment by Tabrarian** [comment]
The merge conflicts are too complex to resolve in the web UI (per the UI tooltip), but I don't have a local repository since I've only been working in the web UI. I'm going to close this one out and try smaller command-line changes to make these fixes. Apologies for the trouble!

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR author is a googler at the time of this PR was created.

**Pull request closed without merging**